rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -79,10 +90,9 @@ void whenNewFederationIsNullThenActiveFederationIsGenesisFederation() {
     }
 
     @Test
-    void whenOldFederationIsNullThenActiveFederationIsNewFederation() {
-        Federation newFederation = getNewFakeFederation(100);
-        when(storageProvider.getNewFederation(federationMainnetConstants, activations)).thenReturn(newFederation);
-        when(storageProvider.getOldFederation(federationMainnetConstants, activations)).thenReturn(null);
+    void getActiveFederation_withNewFederationAndNullOldFederation_returnsNewFederation() {
+        Federation newFederation = createFederation(100);
+        storageProvider.setNewFederation(newFederation);
```

Comment:
And the old federation? According to the test name there should be one

Reply:
Guess we are also missing the test where there is only the new federation and not the old one

Reply:
that is this test, with a "real" new federation and a null old one

Reply:
Looks like I got mixed up, you are right

Reply:
no worries!